.github/workflows/parachain.yml

```diff
@@ -73,28 +73,18 @@ jobs:
         run: rustup show
       # Increase stack limit for beacon light client tests
       - run: sudo prlimit --pid $$ --stack=32768
-      # Run tests for everything except the beacon light client
-      - name: Tests for everything except the beacon light client
+      # Run tests with feature benchmark
+      - name: Tests with benchmark 
         run: >
           cargo test
           --manifest-path parachain/Cargo.toml
           --workspace
           --features runtime-benchmarks
-          --exclude snowbridge-query-events
-          --exclude snowbridge-ethereum-beacon-client
```

Comment:
Remove `Run tests for everything except the beacon light client` it's not necessary.